app/src/main/java/com/jerboa/ui/components/person/PersonProfileLink.kt

```diff
@@ -94,20 +93,13 @@ fun PersonProfileLink(
             }
         }
         if (showTags) {
-            if (isModerator) {
+            if (isDistinguished) {
                 Icon(
                     imageVector = Icons.Outlined.Shield,
```

Comment:
I don't think we should reuse this icon for a different meaning. Isn't there something like a pin icon that we can use?

Reply:
Would the security or localpolice icons work? https://fonts.google.com/icons?selected=Material+Symbols+Outlined:security:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=shield

Reply:
Hmm too similar imo, I was looking for a pin like icon but I couldnt find any at first glance

Reply:
Pins are also already used for stickying posts. Distinguish is kind of a *speak officially* thing. 

Its pry okay to use any shield variant, because I removed showing the mod / admin shields by default with this PR.

Reply:
Ah I was confused what distinguishes means, is it just that the poster has the ability to mark that they are a mod and it should be visible in the comment that they post?

Reply:
Not the poster, but any moderator / admin. They can mark comments where they want to *speak officially* as a moderator.

Reply:
Yean then it's fine